action/staking/withdraw.go

```diff
@@ -42,14 +43,10 @@ func (s Withdraw) Tags() kv.Pairs {
 	}
 	tag2 := kv.Pair{
 		Key:   []byte("tx.owner"),
-		Value: s.From.Bytes(),
-	}
-	tag3 := kv.Pair{
-		Key:   []byte("tx.to"),
-		Value: s.To.Bytes(),
+		Value: s.StakeAddress.Bytes(),
 	}
 
-	tags = append(tags, tag, tag2, tag3)
+	tags = append(tags, tag, tag2)
```

Comment:
Can we add full information to tags(validator address and amount), explorer might need to catch and parse information from this event message and save to Postgresql database.

Reply:
Added